packages/adblocker-puppeteer-example/index.ts

```diff
@@ -59,5 +59,17 @@ function getUrlToLoad(): string {
     console.log('style', style.length, url);
   });
 
+  blocker.on('script-rule-matched', (rule, context) => {
```

Comment:
Should we call them « scriptlet »? I think that’s a more common naming for them.